torch/_decomp/decompositions.py

```diff
@@ -3258,18 +3258,38 @@ def upsample_bilinear2d_aa_vec(input, output_size, align_corners, scale_factors)
     )
 
 
+@register_decomposition(aten.upsample_linear1d.vec)
 @register_decomposition(aten.upsample_bilinear2d.vec)
+@register_decomposition(aten.upsample_trilinear3d.vec)
+@aten.upsample_linear1d.vec.py_impl(DispatchKey.CompositeImplicitAutograd)
+@aten.upsample_linear1d.vec.py_impl(DispatchKey.Autograd)
 @aten.upsample_bilinear2d.vec.py_impl(DispatchKey.CompositeImplicitAutograd)
 @aten.upsample_bilinear2d.vec.py_impl(DispatchKey.Autograd)
-def upsample_bilinear2d_vec(input, output_size, align_corners, scale_factors):
+@aten.upsample_trilinear3d.vec.py_impl(DispatchKey.CompositeImplicitAutograd)
+@aten.upsample_trilinear3d.vec.py_impl(DispatchKey.Autograd)
```

Comment:
I think that just one of them should do? Not sure why they are duplicated

Reply:
I'm not sure either. It's used everywhere in this file.

Reply:
When I remove `CompositeImplicitAutograd`, it fails with
```
RuntimeError: /var/lib/jenkins/workspace/build/aten/src/ATen/RegisterCompositeImplicitAutograd.cpp:5532: SymIntArrayRef expected to contain only concrete integers
```